docs/docs/docs/getting-started/installation.md

```
@@ -387,25 +387,40 @@ You can run the app after closing the terminal or restating the vscode using the
 
-### Importing Sample Data
+### Using the CLI (Dev Containers)
 
 This applies to users running Talawa API in dev containers.
 
 1. Once the server is running, open a new terminal session.
 2. Open a bash session inside the running container:
+
    ```bash
    docker exec -it talawa-api-1 /bin/bash
    ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)